client/src/components/RelatedAndComparison/RelatedAndComparison.jsx

```diff
@@ -1,16 +1,123 @@
-import React from 'react';
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-plusplus */
+import React, { useEffect, useState } from 'react';
 import axios from 'axios';
-import styled from 'styled-components';
-
-const Wrapper = styled.section`
-margin-top: 15em;
-padding: 20em;
-background: #DF9EF6;`;
-
-const RelatedAndComparison = () => (
-  <Wrapper>
-    <h2>Related Products</h2>
-  </Wrapper>
-);
+import RelatedProducts from './assets/RelatedProducts.jsx';
+import YourOutfit from './assets/YourOutfit.jsx';
+
+const RelatedAndComparison = () => {
+  const [relatedIds, setRelatedIds] = useState([]);
+  const [relatedStyles, setRelatedStyles] = useState([]);
+  const [relatedProducts, setRelatedProducts] = useState([]);
+  const [styles, setStyles] = useState([]);
+  const [yourOutfit, setYourOutfit] = useState([]);
+
+  useEffect(() => {
+    axios.get('/api/products/48432/related')
+      .then(({ data }) => {
+        setRelatedIds(data);
+      });
```

Comment:
When you're ready to refactor to useContext, consider replacing the hard-coded '48432'  in the url with state.selectedProduct